python/ray/tests/test_top_level_api.py

```diff
@@ -20,6 +20,7 @@ def test_api_functions():
         "cancel",
         "get_actor",
         "get_gpu_ids",
+        "get_job_submission_client_cluster_info",
```

Comment:
I think this also needs to be removed now that `get_job_submission_client_cluster_info` isn't a top level ray api

Reply:
oh thanks!